programs/vote/src/vote_processor.rs

```diff
@@ -180,48 +180,31 @@ declare_process_instruction!(
             }
             VoteInstruction::UpdateVoteState(vote_state_update)
```

Comment:
next step would be to remove this vote instruction, all votes should be coming over the wire compressed. realized I should've probably had a `!is_active(compact_vote_state_updates)` here.